pttrack/urls.py

```diff
@@ -27,6 +27,9 @@
     url(r'^patient/update/(?P<pk>[0-9]+)$',
         views.PatientUpdate.as_view(),
         name='patient-update'),
+    url(r'^patient/activate/(?P<pk>[0-9]+)$',
+        views.patient_activate,
+        name='patient-activate'),
```

Comment:
A matter of style that I myself violated with the update view is the "directory" structure. I would prefer (although we don't need to do it now) to have it be more like `patient/id/update` and `patient/id/activate` rather than what we have because I feel like the `update` belongs to the patient, not the other way around. Just philosophizing (i.e. no need to act on this) but do you know what I mean?